x-pack/plugin/esql/src/test/java/org/elasticsearch/xpack/esql/expression/function/aggregate/TopTests.java

```diff
@@ -317,4 +317,10 @@ private static TestCaseSupplier makeSupplier(
             );
         });
     }
+
+    @Override
+    protected Expression serializeDeserializeExpression(Expression expression) {
+        // TODO: This aggregation doesn't serialize the Source, and must be fixed.
```

Comment:
It would be really nice to open an issue about it and reference it here from the todo.
This way we could track its progress and know if todos are resolved or accidentally left after the feature was implemented.

Reply:
Sure! That's the issue I was tracking actually, so I should have had one to begin with :hehe: